### PR TITLE
Add network policy support

### LINKED ISSUE #69: [Feature] Add NetworkPolicy to each chart to restrict traffic between pods
**Is this a request for help?**:

No

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
FEATURE REQUEST:
To have better control and secure structure of traffic between each application's pods, every chart has to also deploy a NetworkPolicy descriptor with the matching rules.

- Deny all
- Allow traffic from pods in same namespace on application ports
- Allow traffic from external resources to public endpoint only

**Version of Helm and Kubernetes**:
Any

**Which chart**:
All

### PR DESCRIPTION
**Pull request by danielezer**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Adds support for network policy to all charts

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
fixes #69

### CODE REVIEW

**Comment by eldada** [comment]
CI is failing. Can you verify manually on all charts?
@amithins - can you review JFMC settings?

**Comment by danielezer** [comment]
Sure @eldada.
artifactory-ha: installed successfully
artifactory: installed successfully
xray: installed successfully
distribution: installed successfully
mission-control: installed successfully

**Comment by zendril** [comment]
Not that my LGTM officially counts, but took a look and they look good (to me). Thanks @danielezer

**Comment by eldada** [comment]
@zendril - Every extra pair of eyes is much appreciated! Thx for looking!

File: stable/mission-control/values.yaml [hunk 1]
```diff
@@ -33,6 +33,37 @@ rbac:
...
+  # - name: postgres
+  #   podSelector:
+  #     matchLabels:
+  #       app: postgresql
+  #   ingress:
+  #   - from:
+  #     - podSelector:
+  #         matchLabels:
+  #           app: mission-control
+  # Uncomment to allow only mission-control pods to communicate with elasticsearch (if elasticsearch.enabled is true)
```
Comment (amithins) [inline stable/mission-control/values.yaml]: @danielezer - This is not needed as elasticsearch service is deployed as part of mission-control pod.

Comment (danielezer) [reply stable/mission-control/values.yaml]: Thanks @amithins. Removed

**Pull request merged**